SkylineToolsStore/src/org/labkey/skylinetoolsstore/SkylineToolsStoreController.java

```diff
@@ -1101,9 +1103,27 @@ else if (toolLsid != null &&
                     "Path=/; Domain=;");
             }
 
-            return new HttpRedirectView(
-                AppProps.getInstance().getContextPath() + "/files" + tool.lookupContainer().getPath()
-                + "/" + tool.getZipName());
+            Container toolContainer = ContainerManager.getForId(tool.getContainerId());
+            if (toolContainer == null)
+            {
+                throw new NotFoundException("Tool container not found");
+            }
+
+            org.labkey.api.util.Path path = WebdavService.getPath().append(toolContainer.getParsedPath()).append(FileContentService.FILES_LINK).append(tool.getZipName());
+            WebdavResource resource = WebdavService.get().getResolver().lookup(path);
+            if (resource == null || !resource.isFile())
+                throw new NotFoundException("Resource could not be found: " + path.toString());
+
+            // Add our own 'Content-Disposition' header so that it overwrites the one set in
+            // ResponseHelper.setContentDisposition(HttpServletResponse response, ContentDispositionType type, @NotNull String filename)
```

Comment:
Why not use ResponseHelper.setContentDisposition()?

Reply:
I just caught up with the email thread..   A reference to the issue # would be appropriate.

Reply:
This was added to the the MacCoss issues list: https://www.labkey.org/MacCoss/Issue%20Tracker/issues-details.view?issueId=49580
Added a reference to the issue # in the code.